solr/test-framework/src/java/org/apache/solr/cloud/SolrCloudTestCase.java

```diff
@@ -97,11 +97,9 @@ protected static SolrZkClient zkClient() {
     return cluster.getZkStateReader().getZkClient();
   }
 
-  /** if the system property is not specified, use a random value */
+  /** if the system property is not specified, default to false. The SystemProperty will be set in a beforeClass method. */
   public static boolean isPRS() {
-    return PRS_DEFAULT_PROP == null
-        ? random().nextBoolean()
```

Comment:
oh wow I see how this was the problem, leading to too-little control in consistency over wether it's enabled or not in practice